src/router_hotspot_reputation.erl

```diff
@@ -69,7 +69,6 @@ track_offer(Offer) ->
             ok;
```

Comment:
Can we still track offers when the system is disabled?
That way you can get a sense of things without denying gateways.